src/material/tabs/tab-group.ts

```diff
@@ -566,6 +566,8 @@ export class MatTabGroup extends _MatTabGroupBase {
       defaultConfig && defaultConfig.fitInkBarToContent != null
         ? defaultConfig.fitInkBarToContent
         : false;
+    this.stretchTabs =
+      defaultConfig && defaultConfig.stretchTabs != null ? defaultConfig.stretchTabs : true;
```

Comment:
Why not this?

```ts
defaultConfig?.stretchTabs ?? true
```

Reply:
For me, I was copying the way it was done in the file already, with my thinking that it was the quickest way to get the change approved and merged :)